unlock-app/src/utils/getUserAccountType.ts

```diff
@@ -0,0 +1,14 @@
+import { storage } from '~/config/storage'
+import { UserAccountType } from './userAccountType'
+
+export const getUserAccountType = async (
```

Comment:
This does not add a lot of value and adds complexity. Please use `storage.getUserAccountType(emailAddress)` directly where you need this.